app/main-process/projectWindow.js

```diff
@@ -204,13 +204,19 @@ ProjectWindow.setViewSettingsChanged = function(f) {
 
 ProjectWindow.getViewSettings = function() {
     if(!fs.existsSync(veiwSettingsPath)) {
+        fs.writeFileSync(veiwSettingsPath, JSON.stringify({ theme:'light', zoom:'100' }), {
+            encoding: "utf-8"
+        });
         return { theme:'light', zoom:'100' };
     }
     const json = fs.readFileSync(veiwSettingsPath, "utf-8");
     try {
         return JSON.parse(json);
     } catch(e) {
         console.error('Error in view settings JSON parsing:', e);
+        fs.writeFileSync(veiwSettingsPath, JSON.stringify({ theme:'light', zoom:'100' }), {
+            encoding: "utf-8"
+        });
```

Comment:
Clobbering a file because the JSON syntax is invalid seems a bit too strong. Not sure why it would ever be wrong, but my only guess would be if the user was hand editing it. And I don't think we want to overwrite their file if they missed a curly brace or something.